azure-communication-ui/azure-communication-ui/src/main/res/values-zh-rTW/azure_communication_ui_strings.xml

```diff
@@ -26,7 +26,7 @@
     <string name="azure_communication_ui_setup_view_audio_device_selected_accessibility_label">已選取</string>
     <string name="azure_communication_ui_setup_view_background_gradient_accessibility_label">設定漸層</string>
     <string name="azure_communication_ui_button_switch_camera_accessibility_label">[切換相機] 按鈕</string>
-    <string name="azure_communication_ui_call_setup_action_bar_title">設定</string>
+    <string name="azure_communication_ui_call_setup_action_bar_title">Setup</string>
```

Comment:
Why is this back to English?

Reply:
Touchdown Service Issue. Run it again, and get the translations.